index.html

```diff
@@ -1,30 +1,54 @@
-<head>
-  <link rel="stylesheet" href="sakura.css" type="text/css">
-  <link rel="stylesheet" href="sakura-dark.css" type="text/css">
-</head>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>KISS - Community</title>
+        <meta charset="utf-8">
+        <meta name="description" content="Kiss linux community">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="./assets/css/normalize.css" rel="stylesheet">
+        <link href="./assets/css/skeleton.css" rel="stylesheet">
+        <link href="./assets/css/style.css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
```

Comment:
Could you download Open Sans and include it directly instead of injecting a stylesheet? Should be much smaller that way.

Reply:
done